package.json

```diff
@@ -79,7 +79,7 @@
     "node-gyp": "^8.4.1"
   },
   "engines": {
-    "node": ">=18.19.1",
+    "node": ">=20.15.0",
```

Comment:
CONTRIBUTING.md probably also needs an update, otherwise the first commands mentioned there will fail 😅

Reply:
Yup, done in 1364a7d883b56f5c95bacfd52a86b14e05577037 👍